tests/snapshot/snapshot/restore.yaml

```
@@ -0,0 +1,101 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test _snapshot/{repository}/{snapshot}/_restore endpoints.
+epilogues:
+  - path: /_snapshot/{repository}/{snapshot}
+    method: DELETE
+    status: [200, 404]
+    parameters:
+      repository: my-fs-repository
+      snapshot: my-test-snapshot
+  - path: /_snapshot/{repository}
+    method: DELETE
+    status: [200, 404]
+    parameters:
+      repository: my-fs-repository
+  - path: /restore-index-wait-true
+    method: DELETE
+    status: [200, 404]
+  - path: /restore-index-wait-false
+    method: DELETE
+    status: [200, 404]
+prologues:
+  - path: /_snapshot/{repository}
+    method: PUT
+    parameters:
+      repository: my-fs-repository
+    request:
+      payload:
+        type: fs
+        settings:
+          location: /tmp/opensearch/repo
+  - path: /restore-index-wait-true
```

```suggestion
  - path: /movies
```

NBD, but we tend to use examples like `movies` and `books` because these will appear in the documentation hopefully soon, and we'd like to stay consistent. Consider editing.